recipes/scalene/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set package_name = "scalene" %}
+{% set initial = "s" %}
+{% set version = "1.5.7" %}
+{% set org = "plasma-umass" %}
+{% set author = "emeryberger" %}
+
+package:
+  name: {{ package_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ package_name }}/archive/refs/tags/v{{ version }}.tar.gz
+  # See https://pypi.org/project/scalene/#files
+  sha256: 2d77a1d5f2a1b9314e64e9ec97d068e6e2e0cb74a28f219924477c8b251b806a
+
+build:
+  skip: True  # [py<38]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pip
+  host:
+    - python
+    - setuptools_scm
+    - wheel
+  run:
+    - python
+    - rich
+    - cloudpickle
+    - numpy
+    - pynvml
+
+test:
+  imports:
+    - {{ package_name }}
+  source_files:
+    - test/testme.py
```

Comment:
This file appears to be missing. Should it be included in the recipe as well?

Reply:
It's in the repository... Is that not enough?

Reply:
I just verified that `conda build .` works on my platform and executes that test, as intended.